packages/whosmysanta-backend/.eslintrc.js

```diff
@@ -31,5 +31,10 @@ module.exports = {
     'max-len': ['error', {code: 80}],
     'no-console': 'error',
     'object-curly-spacing': ['error', 'never'],
+    'space-before-function-paren': ['error', {
+      anonymous: 'always', // const foo = function () {}
+      named: 'always', // function foo () {} (Airbnb rule change)
```

Comment:
It would be cool, if we could stick to airbnb as close as possible, any thoughts? :)